UI/NotificationHandler.cs

```diff
@@ -6,25 +6,39 @@
 using Dalamud.Game.Text.SeStringHandling;
 using Dalamud.Game.Text.SeStringHandling.Payloads;
 using Dalamud.Interface;
+using Dalamud.Interface.Windowing;
 using DeathRecap.Events;
 using ImGuiNET;
 
 namespace DeathRecap.UI;
 
-public class NotificationHandler {
+public class NotificationHandler : Window{
     private readonly DalamudLinkPayload chatLinkPayload;
     private readonly DeathRecapPlugin plugin;
     private Death? popupDeath;
     private bool windowWasMoved;
-
-    public NotificationHandler(DeathRecapPlugin plugin) {
+    
+    public NotificationHandler(DeathRecapPlugin plugin) : base("###deathRecapPopup") {
         this.plugin = plugin;
 
+        PositionCondition = ImGuiCond.FirstUseEver;
+        Position = new Vector2(200, 80);
```

Comment:
shouldn't this be
```cs
var viewport = ImGui.GetMainViewport();
var initialPos = new Vector2(
            viewport.WorkPos.X + viewport.WorkSize.X / 2 - 100 * ImGuiHelpers.GlobalScale,
            viewport.WorkPos.Y + viewport.WorkSize.Y / 2 - 40 * ImGuiHelpers.GlobalScale);
```

Reply:
I misread the window size as the window position.
I'll fix that right now.